vivisect/__init__.py

```diff
@@ -2118,8 +2118,59 @@ def makePointer(self, va, tova=None, follow=True):
         if follow and self.isValidPointer(tova):
             self.followPointer(tova)
 
+        # Get type of target and hand into addXref()
+        rflags = 0
+        toloc = self.getLocation(tova)
+        if toloc and toloc[L_LTYPE] == LOC_OP:
+            rflags = envi.BR_PROC
```

Comment:
This seems like an assumption that would not hold true for things like jump tables where the pointer resolves to a LOC_OP but is not actually a function entry point?

Reply:
great catch.  i want it to be marked as a code reference somehow (since the rtype clearly needs to be REF_PTR for `makePointer()`).  iirc (it's been a few months) i put this in but wanted something else, grabbing this as a placeholder.

i was looking to use this for a future vtable-analysis mod, but i'm thinking it may be better to just individually grab pointers next to each other and check the location they point at.
i also have another PR coming, which adds a check for `findPointers()` that does heuristic analysis based on what the pointer points to (`isLikelyPointer()`), to help avoid so many pointers in 0-based firmware.  also, not imperative, since we can just grab the location of the pointed-to-thing.

so, do you think i should choose a different rflag to indicate "code", do a check on the current location to see if it's a `IF_CALL`, or just cut this check out altogether?